web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
I think having a goal of getting properties from one place where default values can be set is good and we should do that.  Making that happen should probably be a separate issue, since this solution matches the rest of the code in ProxyController which is not using GlobalProperties but accessing properties directly from portal.properties.